CMakeLists.txt

```diff
@@ -10,14 +10,16 @@ elseif(ADAPTER STREQUAL "DOOCS")
   set(PROJECT_NAME_PREFIX doocs-)
 elseif(ADAPTER STREQUAL "EPICSIOC")
   set(PROJECT_NAME_PREFIX epics-)
+elseif(ADAPTER STREQUAL "EPICS7IOC")
+  set(PROJECT_NAME_PREFIX epics7-)
 endif()
 
 set(${PROJECT_BASE_NAME}_MAJOR_VERSION 01)
-set(${PROJECT_BASE_NAME}_MINOR_VERSION 01)
+set(${PROJECT_BASE_NAME}_MINOR_VERSION 02)
 set(${PROJECT_BASE_NAME}_PATCH_VERSION 00)
 include(${CMAKE_SOURCE_DIR}/cmake/set_version_numbers.cmake)
 
-project(${PROJECT_NAME_PREFIX}${PROJECT_BASE_NAME}${${PROJECT_BASE_NAME}_MAJOR_VERSION})
+project(${PROJECT_NAME_PREFIX}${PROJECT_BASE_NAME})
```

Comment:
If we have to include the version in the name of the executable, I would like to include the minor version and format it in a way, that it is easily recognizable as a version number.
```suggestion
project(${PROJECT_NAME_PREFIX}${PROJECT_BASE_NAME})
project(${PROJECT_NAME_PREFIX}${PROJECT_BASE_NAME}_${${PROJECT_BASE_NAME}_MAJOR_VERSION}-${${PROJECT_BASE_NAME}_MINOR_VERSION})
```

Reply:
That's exactly the point. It is like "windows 11". It basically is part of the name. And it's not "something-server", but "generic-chimeratk-server". We only put the number extension for this particular generic server, where the server is kind of a "backend" for the configuration, which then is named like the use case. And if I do a config, it is for "generic server 01", and probably incompatible to "generic server 02".

Reply:
If I understand you right, generic-chimeratk-server01 is the one with the DOOCS-backend, generic-chimeratk-server02 might be one with DOOCS- and EPICS-backends, 03 maybe EPICS-backend only, and so on? Basically different versions representing different backend configurations, rather than a development of the application (which would be kinda pointless for an empty app..)?

Reply:
Done